synthtool/gcp/common.py

```diff
@@ -326,6 +326,36 @@ def node_library(self, **kwargs) -> Path:
 
         return self._generic_library("node_library", **kwargs)
 
+    def node_mono_repo_library(self, relative_dir, **kwargs) -> Path:
+        # TODO: once we've migrated all Node.js repos to either having
+        #  .repo-metadata.json, or excluding README.md, we can remove this.
+        if not os.path.exists("./.repo-metadata.json"):
```

Comment:
I believe this check should include `relative_dir` too. I believe the main purpose of this logic is to exclude `samples/README.md` from templates, if there's no `samples/` directory.